code/modules/mob/living/carbon/human/human.dm

```diff
@@ -841,6 +842,37 @@
 			to_chat(usr, "Failed to apply DNA Infusion to [src].")
 			log_admin("[key_name(usr)] failed to apply a DNA Infusion to [key_name(src)].")
 
+	if(href_list[VV_HK_TURN_INTO_MMI])
+		if(!check_rights(R_DEBUG))
+			return
+
+		var/result = input(usr, "This will delete the mob, are you sure?", "Turn into MMI") in list("Yes", "No")
+		if(result != "Yes")
+			return
+
+		if(!ishuman(src))
+			to_chat(usr, "This can only be done to humanoid mobs.")
+			return
```

Comment:
i think this message+check should be before the `input()` is sent to the admin

Reply:
I actually put the check in cargo culted from all the other VV checks on this type. It was put after the input because `input()` is sleeping, so state could change after the user hit yes... But in retrospect I don't think there's a sane world where this proc in `/mob/living/carbon/human/vv_do_topic(list/href_list)` can be run on anything that **isn't** a `/mob/living/carbon/human`. So the check's pretty pointless (as are probably all the other ishuman checks in this this vv_do_topic).